create_db.sh

```diff
@@ -1,9 +1,11 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
This is because /bin/sh is symlinked to /bin/dash right?
I think we should use /bin/bash as this is needed for example by conda.